great-idea-website/index.html

```diff
@@ -15,49 +15,100 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
+	  <header>
+ 	<section class="section1">
+	  <nav>
+	 Services
+  	Product
+  	Vision
+	Features
+  	About
+  	Contact
+	</nav>
+  
 
   <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	  </section>
+  
+	  <section class="section2">
+	  <h1 class="main-header"> 
+ 
+		  <div class="innovation">Innovation</div>    
+		<div class="on">On</div>
+		
+	  	   <div class="Demand">Demand</div>
+		  
+		  <button>
+		Get Started  
+		  
+	</button> 
+		  
+		  </h1>
+
+ 	
+
+  <img src="img/header-img.png" alt="Image-of-a-code-snippet">
+		  </section>
+		  </header>
+<hr>
+	  
+	  <section class="section 3">
+  <h3 class="features">
+	  Features
+	</h3>
+		  <h3 class="about">
+  About
+	</h3>
+  <p class="p-features">
+	Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	  </p>
 
 
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+  <p class="p-about">
+	Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	  </p>
 
   <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
   
-  Services
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
+ <h3 class="services">
+	Services
+	 </h3>
+		  <h3 class="product">
   Product
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	</h3>
+		  <h3 class="vision">
+	  Vision
+	</h3>
+  <p class="p-services">
+	Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	  </p>
```

Comment:
I recommend putting these `h3` & `p` pairs into its own div. Right now, you're making it work really well without it, but if you scrunch up the page, the layout starts breaking _hard_. While responsiveness is not a problem we need to solve in this repo, keep this in mind in the future, for it'll be easier to layout these elements for different viewport sizes with a little more organization.